core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6294,6 +6294,23 @@ private void checkLiteral2(String expression, String expected) {
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBiqQuery));
   }
 
+  @Test public void testTimestampDiffFunctionRelToSql() {
+    final RelBuilder builder = relBuilder();
+    final RexNode timestampDiffRexNode = builder.call(SqlLibraryOperators.TIMESTAMP_DIFF,
+        builder.call(SqlStdOperatorTable.CURRENT_TIMESTAMP),
+        builder.call(SqlStdOperatorTable.CURRENT_TIMESTAMP), builder.literal("HOUR"));
```

Comment:
can we use the third operand as 
builder.literal(TimeUnit.HOUR) instead of builder.literal("HOUR")

Reply:
Done.